torch/onnx/utils.py

```diff
@@ -267,6 +267,27 @@ def _export(model, args, f, export_params=True, verbose=False, training=False,
     return torch_out
 
 
+def _export_module(module, f, operator_export_type=OperatorExportTypes.ONNX, export_type=ExportTypes.PROTOBUF_FILE):
```

Comment:
hm, does it actually support different operator_export_type? from what I understand we don't run the onnx conversion pass here

Reply:
Yeah, we should not be threading these through for export module because the won't work correctly even if set to ONNX